src/Components/Web.JS/src/Boot.WebAssembly.ts

```diff
@@ -116,7 +119,54 @@ async function boot(options?: Partial<WebAssemblyStartOptions>): Promise<void> {
     }
   };
 
+  type BeforeBlazorStartedCallback = (options: Partial<WebAssemblyStartOptions> | undefined, bootConfigResul: BootConfigResult) => Promise<void>;
```

Comment:
```suggestion
  type BeforeBlazorStartedCallback = (options: Partial<WebAssemblyStartOptions> | undefined, bootConfigResult: BootConfigResult) => Promise<void>;
```

Reply:
I've cleaned these up to make them `platform` independent. The caller flows the right set of arguments to them.